Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -591,6 +605,22 @@ public class PBXProjGenerator {
             let resourcesBuildPhase = createObject(id: target.name, PBXResourcesBuildPhase(files: resourcesBuildPhaseFiles))
             buildPhases.append(resourcesBuildPhase.reference)
         }
+        
+        let copyFilesBuildPhasesFiles = getBuildFilesForCopyFilesPhases()
+        if !copyFilesBuildPhasesFiles.isEmpty {
+            for (copyFiles, buildPhaseFiles) in copyFilesBuildPhasesFiles {
+                let copyFilesBuildPhase = createObject(
+                    id: "copy files" + copyFiles.destination.rawValue + copyFiles.subpath + target.name,
+                    PBXCopyFilesBuildPhase(
+                        dstPath: copyFiles.subpath,
```

Comment:
I guess you've tested if a the default `subpath` of `""` is valid, as opposed to nil? As the value is optional in `PBXCopyFilesBuildPhase`. Just trying to minimise diffs when projects are opened in Xcode

Reply:
I’ll reaffirm.

Reply:
The default is `""`. If you don't specify it, it won't exist, which is still valid, but to match what Xcode does naturally it's `""` specified. Xcode won't change when opening either way.

Reply:
👍 
Sometimes Xcode won't change the project on open, but if something else changes (like reordering some files is an easy way to test), then it will rewrite things in it's preferred way